src/main/java/uk/gov/cslearning/catalogue/repository/CourseSuggestionsRepository.java

```diff
@@ -8,4 +8,6 @@
 
 public interface CourseSuggestionsRepository {
```

Comment:
This should extend a PagingAndSortingRepository
public interface CourseSuggestionsRepository extends PagingAndSortingRepository<Course, Integer> {
See 
https://www.baeldung.com/spring-data-jpa-pagination-sorting